### DISCUSSION TITLE
sync group wa error

### DISCUSSION THREAD

**Discussion by newfaboc** [kirimwa-id/forum#24] [Tanya Jawab] [answered]
saya mencoba sync wa group muncul pesan error : 
![image](https://user-images.githubusercontent.com/88919468/129545635-88f41c42-aa62-45d2-ba55-1ef7dff8e139.png)

Namun terkadang berhasil syncronize.

Apakah penyebabnya ?
padahal wa normal dan status wa aktif.
saya coba cek di wa hp juga posisi online.

Terima kasih

**Comment by rioastamal** [comment]
@newfaboc terima kasih feedbacknya. 

Baru saja saya merilis fix di v1.0-RC3 #20 mohon untuk dicoba lagi apakah masih error.

**Comment by newfaboc** [comment, marked as answer]
Saya barusan mencoba untuk merefresh ulang syncronize group. Namun tertulis : 
{"message":"Contacts is being sync.","data":[]}

Saya sudah mencoba merefresh kembali tetap sama. Saya cek status wa di hp online.

Saya juga coba send-message ke personal maupun group juga tidak terkirim.

Apakah ada batas waktu koneksi online di kirimwa.id ?

**Reply by rioastamal** [reply]
> Apakah ada batas waktu koneksi online di kirimwa.id ?

Tidak ada batas waktu. Yang ada batas timeout (waktu tunggu) ketika pesan dikirim. Misal HP anda offline atau tidak terkoneksi maka waktu tunggu adalah 10 detik sebelum pengiriman pesan dibatalkan.

Coba telusuri status output dari pesan yang dikirim `GET /v1/messages/MESSAGE_ID` atau `GET /v1/messages?status=fail` untuk melihat pesan yang gagal terkirim. 

Langkah terakhir jika tetap tidak bisa coba lakukan reconnect dengan scan ulang QR code di `GET /v1/qr?device_id=DEVICE_ID`

**Reply by newfaboc** [reply]
saya telah mencoba scan qr lagi. namun masih mendapat pesan :
{"message":"Contacts is being sync.","data":[]}

saya coba cek get-device : 
{"data":[{"id":"xxx","created_at":"2021-08-12T09:50:02.702Z","status":"connected","connected_at":"2021-08-17T04:10:02.084Z","disconnected_at":"2021-08-17T04:10:01.978Z","disconnected_reason":"Taken over by another WhatsApp Web connection."}],"meta":{"last_key":null}}

apakah ada yang salah ?

note: untuk send message wa personal ke 1 nomor berhasil terkirim.

**Reply by rioastamal** [reply]
> {"message":"Contacts is being sync.","data":[]}

Jika jumlah chat di WhatsApp anda mencapai ribuan, mungkin butuh beberapa detik. Tunggu kira2 beberapa saat hingga satu menit sebelum mengeksekusi `GET /v1/groups` kembali. Tunggu lagi beberapa saat jika masih sinkronisasi.

Pastikan juga tidak ada koneksi aktif dari WhatsApp web lain pada nomer tersebut.

> {"data":[{"id":"xxx","created_at":"2021-08-12T09:50:02.702Z","status":"connected","connected_at":"2021-08-17T04:10:02.084Z","disconnected_at":"2021-08-17T04:10:01.978Z","disconnected_reason":"Taken over by another WhatsApp Web connection."}],"meta":{"last_key":null}}

> apakah ada yang salah ?

Selama tidak ada koneksi lain (atau WhatsApp web lain) yang aktif harusnya tidak masalah.

**Reply by newfaboc** [reply]
sudah behasil syncronize.
Setelah saya mengganti device id dengan yang baru ternyata bisa syncronize. 

note : Namun hari ini (18-08-2021), saya cek tidak bisa lagi. akhirnya saya scan ulang baru berhasil syncronize. kira-kira penyebabnya kenapa kah ?

**Reply by ulilamri419** [reply]
Kalau saya terakhir tidak ada data setelah sync '**{"message":"Contacts is being sync.","data":[]}**', biasanya perlu disconnect - connect device nya. (sesuai info di dokumentasi)
Setelah itu, saya tidak menemukan kendala serupa sampai sekarang (almost 10 days).

Mungkin bisa di pastikan, bahwa "wa-nya" dalam kondisi online terus all the time.

Terima kasih,
Amri
